src/core.c/CompUnit/PrecompilationRepository.pm6

```diff
@@ -230,10 +230,19 @@ Need to re-check dependencies.")
         }
 
         if $resolve {
+            # cw: Will return false if the file is already locked. This now
+            #     implies that the Store must cache its created locking objects
+            #     so that existing objects can be returned, or new ones
+            #     created, upon this check.
+            #
+            #     Actually, this implies two things:
+            #       1) Creation
+            #       2) Lock state check.
             if self.store.destination(
                  $compiler-id,
                  $precomp-unit.id,
-                 :extension<.repo-id>
+                 :extension<.repo-id>,
+                 :lock-full
```

Comment:
Is `:lock-fill` unused?

Reply:
Yeah I see it now

Reply:
Oh, no. Its called `full-lock` elsewhere, but `lock-full` here.

Reply:
Yes, it is used, in [CompUnit::PrecompilationRepository](https://github.com/Xliff/rakudo/blob/rakudo-precomp-nolock2/src/core.c/CompUnit/PrecompilationRepository.pm6#L245)

Reply:
Ah! HOWEVER, the parameters do not match. Am fixing that now. Thanks for the catch, @ugexe!